tests/bluetooth/__init__.py

```diff
@@ -0,0 +1,108 @@
+#  Copyright 2025 Google LLC.
+#  Copyright (c) Microsoft Corporation.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""Shared utilities and constants for Bluetooth BiDi tests."""
+
+from test_helpers import (execute_command, goto_url, send_JSON_command,
+                          subscribe, wait_for_event)
+
+HTML_SINGLE_PERIPHERAL = """
+<div>
+    <button id="bluetooth">bluetooth</button>
+    <script>
+        let device;
+        const options = {filters: [{name:"SomeDevice"}]};
+        document.getElementById('bluetooth').addEventListener('click', async () => {
```

Comment:
Should it be a page script, or maybe it can be evaluated via `script.evaluate`? Then there would be no need in a custom page at all.

Reply:
Done. I removed the custom page and replace it with a new `request_device` utility function.